doc/002-local-dev.md

```diff
@@ -1,184 +1,207 @@
+This produces a `go.work` file.
 
 ```bash
-ziti-controller edge init etc/ctrl.with.edge.yml -u <admin name> -p <admin password>
+$ cat go.work
+go 1.19
+
+use (
+        .
+        ../edge
+)
 ```
 
-Example:
+You must be aware of the checked out revision in each module because it is used to satisfy imports at a higher precedence than pinned versions in `go.mod`. That is, you may change the version of `edge` that is imported by this module at build time by checking out a different version in the `edge` repo.
+
+### Initialize the Controller DB
+
+Before you can run the controller you have to initialize it's database with an administrative user.
 
 ```bash
-ziti-controller edge init etc/ctrl.with.edge.yml -u admin -p o93wjh5n
+ziti-controller edge init ./etc/ctrl.with.edge.yml -u ADMIN_NAME -p ADMIN_PW
 ```
 
-### Starting the Controller
+### Run the Controller
 
-If you wish to start the controller with the Ziti Fabric and Ziti Edge enabled:
+Start the controller with the Ziti Fabric and Ziti Edge enabled (typical):
 
 ```bash
 ziti-controller run etc/ctrl.with.edge.yml
 ```
 
-If you wish to start the Ziti Fabric standalone:
+Alternatively, you may start the controller Ziti Fabric standalone, without the Edge APIs:
 
 ```bash
 ziti-controller run etc/ctrl.yml
 ```
 
-Please note that if you start the controller without the Ziti Edge enabled, the Ziti SDK, and edge router functionality
-will not be usable. The controller can be started and stopped both ways without issue.
+Please note that if you start the controller without the Ziti Edge enabled then Ziti SDK and edge router functionality
+will not be usable. The remainder of this article will assume you're running the controller with Edge enabled.
+
+### Login to the Controller
+
+This step will save a session token in the `ziti` CLI's configuration cache.
+
+```bash
+ziti edge login -u ADMIN_NAME -p ADMIN_PW
+```
+
+Subsequent `ziti` CLI commands will automatically re-use this session token. You'll need to perform this login step again when the token expires.
 
-### Starting  Routers
+### Enroll the Routers
 
-The Ziti Fabric requires at least one router (fabric router or edge router). There are four predefined configuration files 
-for running routers in `etc/` named `001.yml` to `004.yml`. 
+The Ziti Fabric requires at least one router to function. There are four predefined configuration files
+for running routers in `etc/` named `001.yml` to `004.yml`.
 
 Each configuration file refers to certificate and private keys kept in `etc/ca/intermediate/certs` and
-`etc/ca/intermediate/private/`. The steps for starting the router is to first register the router then 
-start it.
+`etc/ca/intermediate/private/`.
 
 Register:
 
 ```bash
-ziti-fabric create router etc/ca/intermediate/certs/XXX-client.cert.pem
+ziti fabric create router etc/ca/intermediate/certs/SERIAL-client.cert.pem
```

Comment:
should be ziti edge create router imo and it's just easier to copy/paste 1-4 so all i need to do is copy one big block

Reply:
and actually - it should just be 1. this is for someone learning how to dev. they don't need 4 routers out of the gate. just givem one. let paul or andrew add permutations/complications like "a full fabric" etc